WireDomain/Sources/WireDomain/Notifications/Builders/Conversation/ConversationUserMessageNotificationBuilder.swift

```diff
@@ -19,7 +19,7 @@
 import WireAPI
 import WireDataModel
 
-struct NewUserMessageNotificationBuilder: NotificationBuilder {
+struct ConversationUserMessageNotificationBuilder: NotificationBuilder {
```

Comment:
I guess this was renamed from another PR, but it still confuses me slightly. I think the structure of the builders could follow the structure of events, so `ConversationEventNotificationBuilder` builds notifications for all conversation events. Internally, there could be separate builders for each events, i.e `NewMessageNotificationBuilder` or `ConversationMemberJoinNotificationBuilder`, et.c